lib/phx_tools_web/live/phx_tools_live/index.ex

```diff
@@ -39,20 +39,23 @@ defmodule PhxToolsWeb.PhxToolsLive.Index do
   end
 
   defp installation_command(live_action) do
-    "$SHELL -c \"$(curl -fsSL #{get_script_url(live_action)})\""
+    "$SHELL -c \"$(curl -fsSL #{get_script_url()})\""
```

Comment:
We don't have to hide it in a different function anymore. Just put the `Endpoint.url()`.

Reply:
Okay right away.